data/json/recipes/recipe_medsandchemicals.json

```diff
@@ -88,10 +88,8 @@
   "difficulty": 2,
   "time": 10000,
   "book_learn": [["adv_chemistry", 1] , ["textbook_chemistry", 1]],
-  "tools": [
-    [
-      [ "chemistry_set", 5 ]
-    ]
+  "qualities": [
+    {"id":"CHEM","level":1}
```

Comment:
Changing electrolysis based recipes was a mistake in the first place, the change being incorrect even with respect to the intent is insult to injury, if you will pardon the breach from my normal formality.